LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -618,9 +620,20 @@ - (void)_dismissMe {
 #pragma mark - UI setup
 - (void)_setupNavBarWithLogoutTitle:(NSString *)logoutTitle {
     // Navigation Bar with custom UI
-    self.navBar =
-    [[UINavigationBar alloc] initWithFrame:CGRectMake(0, LTHMainWindow.frame.origin.y,
-                                                      LTHMainWindow.frame.size.width, 64)];
+    if (LTHiOS11) {
```

Comment:
Usually, it's a good idea to not use conditions if they can be avoided. In this case, we can just use what's in the `if` branch and remove the `if` statement. We can then also remove the `LTHiOS11` macro.